http/src/main/java/com/instancify/scriptify/core/script/function/data/http/HttpRequest.java

```diff
@@ -0,0 +1,63 @@
+package com.instancify.scriptify.core.script.function.data.http;
+
+import okhttp3.*;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class HttpRequest {
+    private final String url;
+    private final String method;
+    private final Map<String, String> headers = new HashMap<>();
+    private String body = "";
+    private String mediaType = "";
+
+    public HttpRequest(String url, String method) {
+        this.url = url;
+        this.method = method;
```

Comment:
Elevate the request method always to upper-case and replace equalsIgnoreCase() with equals() when comparing the method with POST and PUT